mantle/platform/api/do/api.go

```diff
@@ -113,6 +117,9 @@ func (a *API) resolveImage(ctx context.Context, imageSpec string) (godo.DropletC
 	// resolve to user image ID
 	image, err := a.GetUserImage(ctx, imageSpec, true)
 	if err == nil {
+		if image.Type != "custom" {
+			a.opts.IPv6 = true
```

Comment:
We're not handling the numeric image ID case above.

...nor the "magic values" case.  We can assume `true` there.  (It's dead code right now, but might be worth leaving as a placeholder for when we get FCOS official images.  mantle still has quite a few placeholders leftover from Container LInux.)

Reply:
I am setting the default to true then, let me know if where I am setting it makes sense.